lib/sqlite-helper.js

```diff
@@ -3,130 +3,122 @@
  * @module sqlite-helper
  * @author Alexandru Mereacre <mereacre@gmail.com>
  */
+"use strict";
 
-const sqliteConstants = require("./sqlite-constants.js");
 const util = require("util");
+const Promise = require("bluebird");
 
-module.exports = (function() {
-  "use strict";
-
-  const Promise = require("bluebird");
+/**
+ * @global
+ * @typedef {function} SQLiteStatementCreator
+ * @param {Array<string>} dataRowKeys - List of columns in the data.
+ *     This must be in the same order as the data will be binded.
```

Comment:
> as the data will be binded.

Is that a typo or am I missing something?

Reply:
Ah, good shout, let me fix that.

It should be "as the data **that** will be _bound_."